app/models/ownership_certificate.rb

```diff
@@ -2,7 +2,7 @@
 
 class OwnershipCertificate < ApplicationRecord
   belongs_to :planning_application
-  has_many :land_owners
+  has_many :land_owners # rubocop:disable Rails/HasManyOrHasOneDependent
```

Comment:
Same here, planning application is `dependent: :destroy` on `has_one :ownership_certificate`